tests/nb/homeassistant_HassTurnOff.yaml

```diff
@@ -1,12 +1,46 @@
-language: 'nb'
+language: "nb"
 tests:
-- intent:
-    name: HassTurnOff
-    slots:
-      name: "fan.ceiling"
-  sentences:
-  - "slå av tak vifte"
-  - "skru av tak viften"
-  - "slå tak vifte av"
-  - "skru tak viften av"
-  
+  - intent:
+      name: HassTurnOff
+      slots:
+        name: "fan.ceiling"
+    sentences:
+      - "slå av takvifte"
+      - "skru av takviften"
+      - "slå takvifte av"
+      - "skru takviften av"
+  - sentences:
+      - Lukk høyre gardin
+    intent:
+      name: HassTurnOff
+      slots:
+        name: cover.curtain_left
+    response: "høyre gardin ble lukket"
```

Comment:
As I mentioned in a previous comment, this is confusing (left/right naming).

Reply:
Fixed. Thank you.